lib/actions.ts

```diff
@@ -86,23 +87,45 @@ export const getApplication = async (
   }
 };
 
+export const updateApplication = async ({
+  id,
+  isEditorsPick,
+}: {
+  id: string;
+  isEditorsPick: boolean;
+}) => {
+  const state = await retrieveDatabase();
+  const application = getNode(state, id);
+
+  const newState = addNode(state, {
+    ...application!,
+    id,
+    isEditorsPick,
+  });
+
+  await storeDatabase(newState);
+  revalidatePath("/");
+};
+
 export const submitApplication = async ({
   images,
   data,
+  isEditorsPick = false,
```

Comment:
I think we don't need to pass isEditorsPick as an arg here. it is defaulted to `false` when submitting a application anyway

Reply:
You're right! I'll fix it!